handbook/business-operations/README.md

```diff
@@ -135,7 +135,17 @@ Weekly revenue report - [@`todo: CRO` and @`todo: CEO`]
 - Number of customers awaiting invoices = `todo`
 - Number of past-due renewals = `todo`
 ```
+- Send payment reminders via email to all outstanding accounts by responding to the invoice email initially sent to the customer.
+```
+Hello,
+This is a reminder that you have an outstanding balance due for your Fleet Device Management premium subscription.
+We have included the invoice here for your convenience.
+For payment instructions please refer to your invoice, and reach out to billing@fleetdm.com with any questions.
```

Comment:
@JoStableford this PR is failing the "Test Fleet website" due to the billing@ email

Reply:
The test is still failing due to this email. Can we pattern-match what we've done here? https://fleetdm.com/handbook/company/communications#getting-a-contract-signed